### PR TITLE
Add dotted timeline to a single point event

### PR DESCRIPTION
**Pull request by aabdaab**
* If there's only one event on the timeline, keep the dotted line
* All left-aligned as per client's request except when there is only one event in order to show the dotted timeline

**Single event**:

image

**Two events**: 
image

**More events**
image

### CODE REVIEW

File: app/javascript/components/EventsTimeline.js [hunk 1]
```diff
@@ -93,7 +94,7 @@ const EventsTimeline = ({ events, options, isFiltered = false }) => {
         <div ref={eventsContainerEl} onScroll={checkButtons} className="events-container">
           <div className="timeline">
             {currentEvents.map((event) => (
-              <div key={event.title} className="time-point">
+              <div key={event.title} className={cx('time-point', { 'time-point-multiple-events': currentEvents.length > 1 })}>
```
Comment (tsubik) [inline app/javascript/components/EventsTimeline.js]: maybe with BEM modifier `time-point--multiple-events` I know we are not very consistent with using BEM ;)

but after that CSS could be written like
```
.time-pont {
   ....
   &--multiple-events {
   ...
   }
}
```

Comment (aabdaab) [reply app/javascript/components/EventsTimeline.js]: sure, yeah 👍

Comment (tsubik) [reply app/javascript/components/EventsTimeline.js]: actually maybe always could be written like `&-multiple-events` haven't checked that.

**Comment by aabdaab** [comment]
added a commit with nesting ☝️

File: app/assets/stylesheets/cclow/_events_timeline.scss [hunk 2]
```diff
@@ -17,37 +17,37 @@ $arrow-btn-width: 48px;
...
 
-  .time-point-multiple-events {
-    &:last-child {
-      .point {
-        &:after {
-          border: none;
-        }
+    &.time-point-multiple-events {
+      &:first-child {
+        text-align: left;
```
Comment (tsubik) [inline app/assets/stylesheets/cclow/_events_timeline.scss]: this could be moved into first-child of &-multiple-events right?

Comment (tsubik) [reply app/assets/stylesheets/cclow/_events_timeline.scss]: As I can see there is no need to check if both classes are on the element time-point and time-point-multiple-events, hmm

**Comment by aabdaab** [comment]
fixed, gonna merge this

**Pull request merged**